metrics/sourceinfo/sourceinfo_test.go

```diff
@@ -0,0 +1,54 @@
+package sourceinfo
+
+import (
+	"testing"
+)
+
+//TestFindLineNumbers runs unit tests on the linenumbers package
```

Comment:
Let's update this comment to the new package name (or remove the phrase "on the linenumbers package")